dist/windows/strawberry.nsi.in

```diff
@@ -145,13 +145,30 @@ UninstPage custom un.LockedListPageShow
 !insertmacro MUI_UNPAGE_FINISH
 
 !insertmacro MUI_LANGUAGE "English" ;first language is the default language
+!insertmacro MUI_LANGUAGE "Italian"
 
 Name "${PRODUCT_NAME}"
 
+Caption "${PRODUCT_NAME} ${PRODUCT_DISPLAY_VERSION}"
+
 OutFile "${PRODUCT_NAME_SHORT}Setup-${PRODUCT_DISPLAY_VERSION}${build_type}-${compiler}-${arch}.exe"
 
 InstallDir "${PRODUCT_INSTALL_DIR}"
 
+LangString CHECKING_RUNNING_PROGRAMS  ${LANG_ENGLISH} "Checking for running programs:"
+LangString SEARCHING_RUNNING_PROGRAMS ${LANG_ENGLISH} "Searching for running programs..."
+LangString NO_PROGRAMS_TO_CLOSE       ${LANG_ENGLISH} "No programs need to close."
+LangString DOWNLOADING                ${LANG_ENGLISH} "Downloading..."
+LangString ALREADY _INSTALLED         ${LANG_ENGLISH} "${PRODUCT_NAME} is already installed. $\n$\nClick `OK` to remove the previous version or `Cancel` to cancel this upgrade."
```

Comment:
There is a space between `ALREADY` and `_INSTALLED` here.

Reply:
Thanks. Fixed.